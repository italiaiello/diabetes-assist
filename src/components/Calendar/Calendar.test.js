import React from 'react'
import ReactCalendar from './calendar'
import { shallow } from 'enzyme'


it('expects to render the ReactCalendar component', () => {
    expect(shallow(<ReactCalendar />)).toMatchSnapshot()
})