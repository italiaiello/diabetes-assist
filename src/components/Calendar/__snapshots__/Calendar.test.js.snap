// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render the ReactCalendar component 1`] = `ShallowWrapper {}`;
